qp_deblur/deblur.py

```diff
@@ -129,14 +131,31 @@ def deblur(qclient, job_id, parameters, out_dir):
     # Generating artifact
     pb = partial(join, out_dir)
 
+    # Generate the filepaths
+    final_biom = pb('final.biom')
+    final_seqs = pb('final.seqs.fa')
+    final_biom_16s = pb('final.only-16s.biom')
+    final_seqs_na = pb('final.seqs.fa.no_artifacts')
+
+    if not exists(final_biom_16s):
+        # Create an empty table. We need to send something to Qiita that is
+        # a valid BIOM, so we are going to create an empty table
+        t = Table([], [], [])
+        with biom_open(final_biom_16s, 'r+') as f:
```

Comment:
Yeah, I started using r+ when dealing with hdf5 files due to this: http://docs.h5py.org/en/latest/high/file.html#File.mode